sdk/formrecognizer/ai-form-recognizer/src/formTrainingClient.ts

```diff
@@ -352,6 +352,11 @@ export class FormTrainingClient {
     }
   }
 
+  /**
+   * (INTERNAL) Provides an async iterable of model pages.
+   *
+   * @hidden
+   */
```

Comment:
I do not mind having it but it is not needed to tag private members with `@hidden`. I guess you did this because the locally-generated docs still showed them up? This is a known issue that I need to figure out but `docs.microsoft.com` does not show untagged private members.